docs/technical-reference.md

```diff
@@ -250,14 +250,13 @@ User email addresses are stored by the Identity Service in RDS Postgres (part of
 The default Quilt settings are adequate for most use cases. The following section
 covers advanced customization options.
 
-### Setting default role
+### Setting the default role
 
-If your stack settings allow users signing up by themselves, you must set the
-default role which will be assigned to all new users.
-Users won't be able to sign up by themselves until the default role is set,
-though they still can be invited by an admin via user management UI.
+**The Quilt admin must log in and set the default role** in order for new 
+users to be able to log in. Without this, new users will have no permissions.
```

Comment:
@akarve this is not quite correct -- users won't be able to sign up unless default role is set up

Reply:
Fixed.